src/openforms/scss/components/admin/_google-tag-manager.scss

```diff
@@ -0,0 +1,6 @@
+.google-tag-manager {
```

Comment:
not the correct location - this is not an admin component, but a public component (it gets loaded on the form detail page)